models.py

```
@@ -1,15 +1,28 @@
 from __future__ import annotations
 
 from collections.abc import AsyncIterator
-from dataclasses import dataclass
+from datetime import datetime
 from enum import IntEnum
 from typing import Any, ClassVar, Literal
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)